sdk/tests/client/mod.rs

```diff
@@ -270,7 +268,8 @@ fn is_remainder_or_return(output: &Output, amount: u64, address: Address, native
             return false;
         }
 
-        if output.features().len() != 0 {
+        // Can be 1 for a native token
+        if output.features().len() > 1 {
```

Comment:
Maybe we should actually check that the only feature is an NT, just to be extra safe

Reply:
Ok only if native tokens are expected, added a check